youtube_dl/extractor/letv.py

```diff
@@ -76,10 +91,15 @@ def _real_extract(self, url):
             'tkey': self.calc_time_key(int(time.time())),
             'domain': 'www.letv.com'
         }
+
+        self._use_opener('alternative')
```

Comment:
This introduces an implicit global state. Please do not use implicit global state. Instead, a parameter for _download_* or another helper function seems way more reasonable.